wodles/aws/tests/aws_utils.py

```diff
@@ -251,7 +251,7 @@ def get_aws_bucket_parameters(db_table_name: str = TEST_TABLE_NAME, bucket: str
 
 
 def get_aws_service_parameters(db_table_name: str = TEST_TABLE_NAME, service_name: str = 'cloudwatchlogs',
-                               reparse: bool = False,
+                               reparse: bool = False, access_key: str = None, secret_key: str = None,
                                profile: str = TEST_AWS_PROFILE, iam_role_arn: str = None,
```

Comment:
This has been removed [here](https://github.com/wazuh/wazuh/pull/18680/files#diff-3e9d53faf228d4be2432c1e9b9da1b65d119c9c2c93f11960e2987181d2029a1) as part of the removal of the deprecated plain text auth.

Reply:
Fixed in commit e15342e3290b2ea66f47132b432a4b431e1fa4c4 and documented results of tests in https://github.com/wazuh/wazuh/issues/23839#issuecomment-2161536169.